src/core/Akka/Routing/TailChopping.cs

```diff
@@ -122,7 +122,14 @@ public override void Send(object message, IActorRef sender)
                     try
                     {
 
-                        completion.TrySetResult(await (_routees[currentIndex].Ask(message, _within)).ConfigureAwait(false));
+                        completion.TrySetResult(
+                            await (_routees[currentIndex].Ask(message, _within)).ConfigureAwait(false));
+                    }
+                    catch (AskTimeoutException)
+                    {
+                        completion.TrySetResult(
```

Comment:
Ah, is this your fix from your other PR @heatonmatthew ?

Reply:
Yes it is. Happy to remove it if you want them separated?

Reply:
Nope, it's fine to keep it in. Just wanted to make sure I understood what it was.